### DISCUSSION TITLE
How to embed PDF documents with meta data?

### DISCUSSION THREAD

**Discussion by jonison-zhang** [milvus-io/milvus#30602] [Ideas & Feature requests]
Hi, there, a newbie of using Milvus data base. I can perform embedding pdf documents successfully, but would like to know how to get meta data for each embeded content. The first page of each pdf contains the title of document, author and affiliation. Is there any way that can embed the first page as the meta data which will be retrieved via Langchian, etc. meta data shows where the answers is from?

many thanks,

**Comment by yhmo** [comment]
You'd better go to the Langchain community to ask this question.
